seed/challenges/object-oriented-and-functional-programming.json

```diff
@@ -236,7 +236,7 @@
         "filter is a useful method that can filter out values that don't match a certain criteria",
         "Let's remove all the values greater than five",
         "<code>array = array.filter(function(val) {</code>",
-        "<code>&thinsp;&thinsp;return val<4;</code>",
+        "<code>&thinsp;&thinsp;return val <= 5;</code>",
```

Comment:
I agree with @abhisekp. That said, these are both minor fixes and they both look good, so I'm going to go ahead and merge. Thanks for your contributions!